packages/commons-test/src/testUtils.ts

```diff
@@ -164,11 +164,17 @@ export const getMockAgreementAttribute = (
 export const getMockEServiceAttributes = (num: number): EServiceAttribute[] =>
   new Array(num).map(() => getMockEServiceAttribute());
 
-export const getMockEService = (
-  eserviceId: EServiceId = generateId<EServiceId>(),
-  producerId: TenantId = generateId<TenantId>(),
-  descriptors: Descriptor[] = []
-): EService => ({
+export const getMockEService = ({
```

Comment:
`feature/incaricato` is already super big and impactful 🥲 
I know our mocks could be improved (a lot, especially in terms of DX), but I would avoid a DX refactor in this feature branch that has consequences on many tests.
I would just add the `isDelegable` as the last optional param and add it only in new usages, or just use the `getMockEService` as it is and add `isDelegable` only in tests where it's needed, for example:

```ts
const delegableEservice = {
  ...getMockEService(),
  isDelegable: true
}
```